htdocs/core/class/html.form.class.php

```diff
@@ -693,7 +693,10 @@ public function selectMassAction($selected, $arrayofaction, $alwaysvisible = 0)
 
 		$ret .= '</select>';
 
-		if (empty($conf->dol_optimize_smallscreen)) $ret .= ajax_combobox('.massactionselect');
+		if (!empty($conf->use_javascript_ajax))
```

Comment:
This should be useless because if !empty($conf->use_javascript_ajax)) is false, then ajax_combobox should return empty string.

Reply:
in v12 alfa if java inactive i have an erreur

Reply:
So there is a trouble somewhere else. What is your error ?